components/cstudio-contextual-nav/wcm-site-dropdown-mods/wcm-root-folder.js

```diff
@@ -232,36 +233,51 @@
                         pathLength = 1;
                     }
 
-                    for (var i = 0; pathLength > i; i++) {
-                        var servPath;
-                        if (Object.prototype.toString.call(rootPath) === '[object Array]') {
-                            servPath = rootPath[i];
-                        } else {
-                            servPath = rootPath;
-                        }
-                        CStudioAuthoring.Service.lookupSiteContent(site, servPath, 1, "default", {
-                            openToPath: pathToOpen,
-                            success: function (treeData) {
+                    var ind=0;
+                    var servPath;
+                    if (Object.prototype.toString.call(rootPath) === '[object Array]') {
+                        servPath = rootPath[ind];
+                    } else {
+                        servPath = rootPath;
+                    }
 
-                                //if(servPath == "/site/website")
-                                window.treeData = treeData;
+                    (function (ind) {
+                        function treePrint(servPath) {
+                            CStudioAuthoring.Service.lookupSiteContent(site, servPath, 1, "default", {
+                                openToPath: pathToOpen,
+                                success: function (treeData) {
 
-                                var items = treeData.item.children;
-                                if (instance.showRootItem) {
-                                    items = new Array(treeData.item);
-                                }
-                                Self.drawTree(items, tree, path, instance, pathFlag);
-                                pathFlag = false;
-                                YDom.removeClass(label, "loading");
-                                //add hover effect to nodes
-                                Self.nodeHoverEffects(this);
-                            },
+                                    //if(servPath == "/site/website")
```

Comment:
always remove code rather than comment it out.  If it needs a comment why it was removed, add it to the source control